packages/component/src/Localization/pt-BR.js

```diff
@@ -1,3 +1,44 @@
+function xMinutesAgo(dateStr) {
+  const date = new Date(dateStr);
+  const dateTime = date.getTime();
+
+  if (isNaN(dateTime)) {
+    return dateStr;
+  }
+
+  const now = Date.now();
+  const deltaInMs = now - dateTime;
+  const deltaInMinutes = Math.floor(deltaInMs / 60000);
+  const deltaInHours = Math.floor(deltaInMs / 3600000);
+
+  if (deltaInMinutes < 1) {
+    return 'Agora a pouco';
+  } else if (deltaInMinutes === 1) {
+    return 'Um minuto atrás';
+  } else if (deltaInHours < 1) {
+    return `${ deltaInMinutes } minutos atrás`;
+  } else if (deltaInHours === 1) {
+    return `Uma hora atrás`;
+  } else if (deltaInHours < 5) {
+    return `${ deltaInHours } horas atrás`;
+  } else if (deltaInHours <= 24) {
+    return `Hoje`;
+  } else if (deltaInHours <= 48) {
+    return `Ontem`;
+  } else if (window.Intl) {
+    return new Intl.DateTimeFormat('pt-BR').format(date);
+  } else {
+    return date.toLocaleString('pt-BR', {
+      day: '2-digit',
+      hour: '2-digit',
+      hour12: false,
+      minute: '2-digit',
+      month: '2-digit',
+      year: 'numeric',
+    });
+  }
+}
+
 export default {
   // FAILED_CONNECTION_NOTIFICATION: '',
```

Comment:
Are you able to fill the commented-out lines to finish up this localization file? :)

Reply:
Sure!